src/commands/distribute/stores/list.ts

```diff
@@ -0,0 +1,48 @@
+import { AppCommand, CommandResult, ErrorCodes, failure, help, success } from "../../../util/commandline";
+import { AppCenterClient, models, clientRequest, ClientResponse } from "../../../util/apis";
+import { out } from "../../../util/interaction";
+import { inspect } from "util";
+import * as _ from "lodash";
+
+const debug = require("debug")("appcenter-cli:commands:distribute:stores:list");
+
+@help("Lists all stores of the app")
```

Comment:
The help for apps list is "Get list of configured applications"
Let's make this one something like "Get list of stores configured for an app"